doc/Type/Supply.pod6

```diff
@@ -620,6 +620,22 @@ C<:expires> parameter how long to wait (in seconds) before "resetting"
 and not considering a value to have been seen, even if it's the same as
 an old value.
 
+=head2 method repeated
+
+    method repeated(Supply:D: :&as, :&with)
+
+Creates a supply that only provides repeated values, as defined
+by the optional C<:as> and C<:with> parameters (same as with
+L<C<unique>|/language/independent-routines#routine_unique>).
+
+    my $supply = Supply.from-list(<a A B b c b C>).repeated(:as(&lc));
+    $supply.tap(&say);           # OUTPUT: «A␤b␤b␤C␤»
+
+See L<C<repeated>|/language/independent-routines#routine_repeated>
+for more examples that use its sub form.
+
+B<Note>: Available since version 6.e (2020.01 and later).
```

Comment:
Maybe we should specify it is "Rakudo 2020.01 and later"?

Reply:
I pushed a new commit with 'Rakudo' specified. Thanks !